pkgs/test_api/lib/src/backend/declarer.dart

```diff
@@ -143,6 +158,12 @@ class Declarer {
       bool solo = false}) {
     _checkNotBuilt('test');
 
+    final fullName = _prefix(name);
+    if (_filterToSingleTestName != null &&
```

Comment:
Do you need the null check here? (is fullName nullable?)

Reply:
No, `_prefix` returns non-nullable String.